docs/platforms/apple/guides/ios/session-replay/customredact.mdx

```diff
@@ -75,3 +75,28 @@ To hide the username, we need to mask it.
     .sentryReplayUnmask() 
   }
 ```
+
+## Debugging session replay masking
+
+To visualize which elements are being masked, you can enable a masking overlay.
+
+Call the following method to display the red masking overlay on all masked views:
+
+```swift
+  SentrySDK.replay.showMaskingOverlay()
+```
+
+You can preview masking during the design phase of your SwiftUI views using the `sentryReplayPreviewMask` modifier. This helps ensure proper masking without affecting the final release build.
+
+To avoid accidentally including it in production, apply the modifier only in your preview code:
+
+```swift
+struct ContentView_Previews: PreviewProvider {
+  static var previews: some View {
+    ContentView()
+      .sentryReplayPreviewMask()
+  }
+}
+```
+
+This approach allows you to fine-tune masking behavior while designing your UI, ensuring sensitive information is properly protected in production.
```

Comment:
```suggestion
```

Reply:
`m`: I'd remove this or merge this with the text above since it's almost the same content